internal/formats/common/cyclonedxhelpers/properties.go

```diff
@@ -63,6 +68,8 @@ func getCycloneDXPropertyValue(field reflect.Value) string {
 		return ""
 	case reflect.Ptr:
 		return getCycloneDXPropertyValue(reflect.Indirect(field))
+	case reflect.Struct:
```

Comment:
We should move this up to line 65 instead. Having the `field.CanInterface()` check is important to avoid potential panics during reflection.

Reply:
I added `canInterface` check for filed.